package.json

```diff
@@ -96,6 +96,9 @@
     "typechain": "^5.0.0",
     "typescript": "^4.2.2"
   },
+  "peerDependencies": {
+    "jsbi": "3.2.5"
```

Comment:
Maybe this should be "^3.2.5", otherwise we are requiring the peer has exactly 3.2.5 which is probably too strict/restrictive?